module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -1365,20 +1405,22 @@ public function getMyProfile($patron)
      *                         was checked out (optional – introduced in release 2.4)
      *
      * @param array $patron Patron login information from $this->patronLogin
+     * @param array $params Additional parameters (limit, page, sort)
      *
      * @return array Transactions associative arrays
      */
-    public function getMyTransactions($patron)
+    public function getMyTransactions($patron, $params = [])
     {
-        $query = ['query' => 'userId==' . $patron['id'] . ' and status.name==Open'];
+        $limit = $params['limit'] ?? 1000;
```

Comment:
My thought was that if we don't get a limit value, we should assume that pagination is not set up correctly and try to display all the data. I think the limit of 1000 was intended to match the functionality that existed before we added pagination, though I may be misremembering since it's already been a while since I did this!

Reply:
Ah ok that makes sense!